README.md

```diff
@@ -213,7 +213,7 @@ When needed, you can also store data in more complicated ways. For instance, may
 
 ```ruby
 # Create an array (list of things of any types)
-grocery_list = ['Potatoes', 'Bread', 'Milk', 'Butter']
+grocery_list = ["Potatoes", "Bread", "Milk", "Butter"]
```

Comment:
Oh, I also changed these to double quotes, since if you tried this code in irb Ruby will give you them back in double quotes (regardless of how the original strings were entered). I made this change to students don't get confused as to why you might enter one thing and get another thing back.